lib/src/drag_select_grid_view/drag_select_grid_view.dart

```diff
@@ -38,7 +38,7 @@ typedef SelectableWidgetBuilder = Widget Function(
 ///
 /// A long-press enables selection. The user may select/unselect any item by
 /// tapping on it. Dragging allows cascade select/unselect.
-/// The above behaviour can be altered by enabling single click selection.
+/// The above behaviour can be altered by enabling [triggerSelectionOnTap].
```

Comment:
Just a detail: Those docs are written in american english, so I believe we should spell "behavior" instead of "behaviour" :)